### ISSUE TITLE
Crash on server startup Pollen 1.4.3

### ISSUE THREAD

**Issue by BugmanBugman** [MoonflowerTeam/pollen#46]
### What version are you seeing the problem on?

1.18.2

### Describe the issue

```markdown
1. start up a server
2. wait til it loads
3. then the server will suddenly crash
```

### latest.log

```shell
https://gist.github.com/BugmanBugman/981a8f195116ded9685cc25d3388a55b
```

**Comment by mintynoura** [comment]
i'm having the same problem as well. I couldn't recreate it with only pollen installed, but removing it from my modlist made everything work fine. It might be a compatability issue

**Comment by tazzy989** [comment]
Hi, this may not be the exact same issue but I'm getting this screen:
"Errors in currently selected datapacks prevented the world from loading." And referred to return to title or use safe mode. 

I've narrowed it down to a compatibility issue with Placebo 6.4.0 and Pollen 1.4.3, as these are the only two mods I have installed, and I'm still getting the error.

Edit for clarity: this is when attempting to start up an already created world, or to create a new world, both single-player.
